packages/neo-one-website/tutorial/tutorial.md

```diff
@@ -68,6 +68,8 @@ npm install @neo-one/cli @neo-one/client @neo-one/smart-contract @neo-one/smart-
 
 We recommend taking a moment to [setup your editor](/docs/environment-setup#Editor-Setup) to take advantage of inline NEO•ONE compiler diagnostics.
 
+5. Review the available [configuration options](/docs/config%20options) and make the necessary modifications to the `.onerc` file.
```

Comment:
Maybe note that they may want to change `framework` to `react` if they want to follow along with the tutorial since it leverages react and the default is `none`.

Reply:
Also see above note about the slug. `config-options`